test/steps/data.go

```diff
@@ -99,10 +106,14 @@ func (data *Data) AfterScenario(pickle *messages.Pickle, err error) {
 		}
 		return nil
 	})
+	if error != nil{
```

Comment:
This should be moved to the end of function, so steps below are executed even in case of error above.